Lib/test/test_time.py

```diff
@@ -64,9 +65,13 @@ def test_time(self):
         self.assertTrue(info.adjustable)
 
     def test_clock(self):
-        time.clock()
+        with support.check_warnings(('time.clock has been deprecated',
```

Comment:
Why not use `self.assertWarns()`? I think `support.check_warnings()` is for 2.7 code.

Reply:
I didn't know assertWarns(). Why do we still have support.check_warnings()? check_warnings() docstring doesn't mention assertWarns(). It's still widely used in the test suite:

```
haypo@selma$ grep check_warnings Lib/test/test*py|wc -l
115
```

Anyway, I modified my PR to use assertWarns().

Reply:
Maybe all this is inherited from Python 2. Or other developers don't know assertWarns(). Or check_warnings() can be used in the cases when assertWarns() can't (not this case).